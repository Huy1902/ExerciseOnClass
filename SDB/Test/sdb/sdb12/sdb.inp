70
54
-59
71
84
60
85
54
84
-42
62
64
38
43
29
89
5
96
27
38
47
79
81
52
47
56
39
92
83
7
33
8
64
2
36
83
95
77
14
87
87
97
47
79
80
46
52
78
67
11
72
63
58
14
53
94
75
52
10
41
47
26
18
77
73
45
21
56
13
1
32
