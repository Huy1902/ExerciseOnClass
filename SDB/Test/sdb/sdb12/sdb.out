43
-59 
71 
60 
85 
-42 
62 
43 
29 
89 
5 
96 
27 
81 
39 
92 
7 
33 
8 
2 
36 
95 
97 
80 
46 
78 
67 
11 
72 
63 
58 
53 
94 
75 
10 
41 
26 
18 
73 
45 
21 
13 
1 
32 
